src/components/articleStory/ArticleStoryTile.jsx

```diff
@@ -0,0 +1,27 @@
+import "./ArticleStoryTile.css"
+
+function ArticleStoryTile (props) {
+    if (props.imageUrl === "null")
```

Comment:
Het is netter om hier een ternary operator te gebruiken i.p.v. een if/else statement.

{voorwaarde ? is de voorwaarde true dan laat dit zien : is de voorwaarde false laat dan dit zien}

{ props.imageUrl === "null" ? <p> laat mij zien </p> : <p> laat anders mij zien</p>}